source/jormungandr/tests/chaos_disruptions_tests.py

```diff
@@ -506,13 +501,20 @@ def test_disruption_on_stop_point(self):
         disruptions = response.get('disruptions')
         assert not disruptions
 
+        journey_query = journey_basic_query + "&data_freshness=base_schedule"
+        response = self.query_region(journey_query)
+        is_valid_journey_response(response, self.tester, journey_query)
```

Comment:
Maybe add a check on the number of journeys? Seeing that pt is ok for base, and back for adapted (next test).